rockhound/slab2.py

```diff
@@ -0,0 +1,141 @@
+
+    Slab2 is a three-dimensional compilation of global subduction geometries (depth,
+    dip, strike and thickness), separated into regional models for each major
+    subduction zone.
+    Each model is based on a probabilistic non-linear fit to data from a combined
+    catalog consisting of several independent data sets - historic earthquake
+    catalogs, CMT solutions, active seismic profiles, global plate boundaries,
+    bathymetry and sediment thickness information [BEDMAP2]_.
+
+    Parameters
+    ----------
+    zone : str
+        Subduction zone to fech the model.
+        Available zones:
+            - `alaska`: Alaska
+            - `calabria`: Calabria
+            - `caribbean`: Caribbean
+            - `cascadia`: Cascadia
+            - `central_america`: Central America
+            - `cotabalo`: Cotabalo
+            - `halmahera`: Halmahera
+            - `hellenic`: Hellenic Arc
+            - `himalaya`: Himalaya
+            - `hindu_kush`: Hindu Kush
+            - `izu_bonin`: Izu-Bonin
+            - `kamchatka`: Kamchatka-Kuril Islands-Japan
+            - `kermadec`: Kermadec
+            - `makran`: Makran
+            - `manila_trench`: Manila Trench
+            - `muertos_trough`: Muertos Trough
+            - `new_guinea`: New Guinea
+            - `pamir`: Pamir
+            - `philippines`: Philippines
+            - `puysegur`: Puysegur
+            - `ryukyu`: Ryukyu
+            - `scotia_sea`: Scotia Sea
+            - `solomon_islands`: Solomon Islands
+            - `south_america`: South America
+            - `sulawesi`: Sulawesi
+            - `sumatra_java`: Sumatra-Java
+            - `vanuatu`: Vanuatu
+    load : bool
+        Wether to load the data into an :class:`xarray.Dataset` or just return the
+        path to the downloaded data. If False, will return a list with the paths to the
+        subduction grids, respectively.
+    kwargs
+        Keyword arguments will be forwarded to the :func:`xarray.open_dataset` function
+        that loads the grid into memory.
+
+    Returns
+    -------
+    grid : :class:`xarray.Dataset` or str
+        The loaded grid or the file path to the downloaded data.
+    """
+    if zone not in ZONES:
+        raise ValueError("Invalid slab zone: {}".format(zone))
+    fnames = [
+        REGISTRY.fetch(
+            "{}_slab2_{}.grd".format(ZONES[zone]["fname_indicator"], dataset)
+        )
+        for dataset in DATASETS
+    ]
+    if not load:
+        return fnames
+    arrays = [xr.open_dataarray(f).rename(x="longitude", y="latitude") for f in fnames]
+    for array, dataset in zip(arrays, DATASETS):
+        array.name = dataset
+        # Change long_name and add units of each array
+        array.attrs["long_name"] = DATASETS[dataset]["name"]
+        array.attrs["units"] = DATASETS[dataset]["units"]
+    # Merge arrays into a single xr.Dataset
+    ds = xr.merge(arrays)
+    # Change units of thickness, depth and depth_uncertainty to meters
```

Comment:
We also need to modify the units of the `actual_range` attribute of the `thickness`, `depth` and `depth_uncertainty`. They are currently in the original units (km) what does not agree with the actual units of the data.

Reply:
I resolved this!